Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp

```diff
@@ -1009,6 +1013,16 @@ MediaPlayer::SupportsType MediaPlayerPrivateGStreamerMSE::supportsType(const Med
     return extendedSupportsType(parameters, result);
 }
 
+bool MediaPlayerPrivateGStreamerMSE::supportsFeatures(const String& features)
+{
+    // Apple TV requires this one for DD+
+    constexpr auto dolbyDigitalPlusJOC = "joc";
+    if (features == dolbyDigitalPlusJOC)
```

Comment:
Would it make sense to convert the features string to lowercase before doing the comparison? My understanding is that `audio/mp4; codecs="mp4a.40.2"; features="JOC"` wouldn't match, but it should, according to the description of the pull request.

Reply:
Lowercase is guaranteed by https://github.com/WebPlatformForEmbedded/WPEWebKit/blob/154984fd4340d566d2b5494a9ad4fe137c9f50ad/Source/WebCore/Modules/mediasource/MediaSource.cpp#L847 at the moment, but if you think we can sacrifice tiny bit of performance and improve readability here, I can add conversion to lowercase in this function.

Reply:
No, not needed, that initial lowercase conversion you mentioned is enough.